db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/StateMachineMethods.class/__methods__/task_finished.rb

```diff
@@ -9,10 +9,12 @@
 case $evm.root['vmdb_object_type']
 when 'service_template_provision_task'
   final_message += "Service [#{@task.destination.name}] Provisioned Successfully"
+  $evm.create_notification(:type => :automate_service_provisioned, :subject => @task.destination) if @task.miq_request_task.nil?
 when 'miq_provision'
   final_message += "VM [#{@task.get_option(:vm_target_name)}] "
   final_message += "IP [#{@task.vm.ipaddresses.first}] " if @task.vm && !@task.vm.ipaddresses.blank?
   final_message += "Provisioned Successfully"
+  $evm.create_notification(:type => :automate_vm_provisioned, :subject => @task.vm)
```

Comment:
@tinaafitz when provisioning a service with multiple vms will there be multiple notifications be created by this code?